packages/nextjs-mf/src/plugins/JsonpChunkLoading.ts

```diff
@@ -28,27 +33,37 @@ function getCustomJsonpCode(
   return Template.asString(code);
 }
 
+/**
+ * CustomWebpackPlugin class.
+ */
```

Comment:
The 'getCustomJsonpCode' function is quite complex. Consider breaking it down into smaller, more manageable functions. This would improve readability and maintainability.